solidity-v1/dashboard/src/components/modal/wallets/TallyModal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { ModalHeader } from "../Modal"
+import { SelectedWalletModal } from "./SelectedWalletModal"
+import * as Icons from "../../Icons"
+import { withBaseModal } from "../withBaseModal"
+
+const rejectedConnectionErrorMsg =
+  "You rejected the connection request in Tally. Please close and try again, and click confirm connection in Tally window."
```

Comment:
I think too many `and`a are used here 😅 I would change it to:

```suggestion
  "You rejected the connection request in Tally. Please close and try again, then click confirm connection in Tally window."
```

Thoughts?

Reply:
If you agree then also change the same thing in `MetaMaskModal.jsx`